zellij-utils/README.md

```diff
@@ -0,0 +1,6 @@
+# zellij-utils
+
+Utility macros for how to read KDL-files(layout & other config). Also holds types like [Event](https://docs.rs/zellij-utils/latest/zellij_utils/data/enum.Event.html) which is how a zellij plugin receives information from the zellij session.
+
+Generated documentation can be found at [crates.io](https://docs.rs/zellij-utils/latest/zellij_utils/)
+
```

Comment:
I don’t know how github render files but this one is the only one that as a final blank line. The POSIX definition of a line stipulate that it ends with a newline character. => Not sure if this file have 2 or a newline is missing in the other readmes ^^'